source/FilesServer/app/service.py

```diff
@@ -15,3 +16,11 @@ async def s3_download(file_name: str) -> str:
         ExpiresIn=settings.presigned_url_expiration,
     )
     return presigned_url
+
+
+async def s3_upload(file: UploadFile = File(...)) -> None:
```

Comment:
i think we can simplify the type annotation to:
...(file: File) -> ....

Reply:
fixed